scripts/setup_macos_dolphin.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+"""
+Setup script for macOS M-series (Apple Silicon) Dolphin compatibility.
+
+The bundled Dolphin has Python 3.13.5 embedded but is missing the standard library.
+This script downloads Python 3.13.5 from python.org and copies the stdlib into each
+Dolphin instance's framework bundle.
+
+Run this after clone_dolphins.py and before running BTR.py or BTR_test.py on macOS.
```

Comment:
If we decide to keep the script tho, I think we can call this before `clone_dolphins.py` and then clone dolphin and its bundles.